tests/functions.py

```diff
@@ -305,3 +309,18 @@ def ping_host(host):
         return False
     else:
         return True
+
+
+def wait_on_job(job_id):
+    global job_results
+    timeout = 0
+    while True:
+        job_results = GET(f'/core/get_jobs/?id={job_id}')
+        job_state = job_results.json()[0]['state']
+        if job_state in ('RUNNING', 'WAITING'):
+            sleep(5)
+        elif job_state == 'SUCCESS' or job_state == 'SUCCESS':
```

Comment:
This is redundant.
```suggestion
        elif job_state == 'SUCCESS':
```

Reply:
Yeah the second is supposed to be 'FAILED'

Reply:
Ah.  In that case, I find the "in" easier to read:
```suggestion
        elif job_state in ('SUCCESS', 'FAILED'):
```

Reply:
Yeah that what I had in mind thanks